sklearn/feature_extraction/tests/test_text.py

```diff
@@ -892,6 +892,17 @@ def test_countvectorizer_vocab_dicts_when_pickling():
         assert_equal(cv.get_feature_names(), unpickled_cv.get_feature_names())
 
 
+def test_countvectorizer_string_object_as_input():
+    # Ensure that string object is not given as input.
```

Comment:
I thought its enough as other comments in the file are also kind of like this. If you want I'll make an edit.